frontend/src/CreateShipment.js

```diff
@@ -142,17 +169,60 @@ const FormStyled = styled.form`
 `
 
 const LabelStyled = styled.label`
-  color: #fff;
-  font-size: 1.25rem;
-  font-weight: bold;
-  margin-left: 0.5em;
+  font-family: 'Roboto', sans-serif;
+  font-size: 0.75rem;
+  font-weight: 500;
+  letter-spacing: 0.3px;
+  color: #222222;
```

Comment:
Ich glaube du verwendest einige Farben mehrfach, die könntest du auch in den global styles als variablen definieren und dann mehrfach verwenden.

Reply:
gute Idee, schon erledigt.